actioncable/lib/action_cable/helpers/action_cable_helper.rb

```diff
@@ -20,9 +20,17 @@ module ActionCableHelper
       # Make sure to specify the correct server location in each of your environments
       # config file:
       #
-      #   config.action_cable.url = "ws://example.com:28080"
-      def action_cable_meta_tag
-        tag "meta", name: "action-cable-url", content: Rails.application.config.action_cable.url
+      #   config.action_cable.mount_path = "/cable123"
+      #   <%= action_cable_meta_tag %> would render:
+      #   => <meta name="action-cable-url" content="/cable123" />
+      #
+      #   config.action_cable.mount_path = nil
+      #   <%= action_cable_meta_tag(url: 'ws://example.com:28080') %> would render:
+      #   => <meta name="action-cable-url" content="ws://example.com:28080" />
+      #
+      #
+      def action_cable_meta_tag(url:)
+        tag "meta", name: "action-cable-url", content: Rails.application.config.action_cable.mount_path
```

Comment:
Ahh -- I think I misunderstood your previous comment. Also, as apart of this PR, we're retiring `config.action_cable.url` in favor of `config.action_cable.mount_path`, so I simplified your `content:` a little :heart: